ext/ITensorNetworksEinExprsExt/src/ITensorNetworksEinExprsExt.jl

```diff
@@ -0,0 +1,67 @@
+module ITensorNetworksEinExprsExt
+
+using ITensors
+using ITensorNetworks
+using ITensorNetworks: Index
+using EinExprs: EinExprs, EinExpr, einexpr, SizedEinExpr
+
+function EinExprs.einexpr(tn::ITensorNetwork; optimizer::EinExprs.Optimizer)
+  IndexType = Any
+  T = vertextype(tn)
+
+  tensors = EinExpr{IndexType}[]
+  tensor_map = Dict{Set{IndexType},T}()
+  sizedict = Dict{IndexType,Int}()
+
+  for (key, tensor) in pairs(vertex_data(tn))
+    _inds = collect(inds(tensor))
```

Comment:
```suggestion
    inds_v = collect(inds(tensor))
```
to indicate it is the indices of the tensor on vertex `v`. I don't really like using variable names that start with `_`.